docs/source/locale/uk/LC_MESSAGES/tutorial.po

```diff
@@ -101,6 +101,19 @@ msgstr "Організатор може редагувати аукціон ті
 msgid "When this period ends 403 error will be returned on editing attempt:"
 msgstr "Після закінчення цього періоду на спробу редагування буде повернуто помилку 403: "
 
+msgid "Organizer can provide custom *enquiryPeriod.endDate*, and this date should be at least 5 days before *tenderPeriod.endDate*."
+msgstr "Організатор має можливість встановити власний *enquiryPeriod.endDate*, але дата повинна бути не пізніша, ніж за 5 днів до *tenderPeriod.endDate*."
+
+msgid "If *enquiryPeriod.endDate* provided by Organizer is closer to *tenderPeriod.endDate* than 5 days "
+"`422 Unprocessable Entity` code will be returned with error message "
+"'enquiryPeriod.endDate should be not less than 5 days before tenderPeriod.endDate'."
```

Comment:
`the difference between enquiryPeriod.endDate and tenderPeriod.endDate shoul not be less than 5 working days`
Try this variant. It`s rather clear for the general comprehension